pkg/meta/base.go

```diff
@@ -280,57 +281,56 @@ func (m *baseMeta) groupBatch(batch map[Ino]dirStat, size int) [][]Ino {
 	return batches
 }
 
-func (m *baseMeta) calcDirStat(ctx Context, ino Ino) (space, inodes uint64, err error) {
+func (m *baseMeta) calcDirStat(ctx Context, ino Ino) (length, space, inodes uint64, err error) {
```

Comment:
we should use `dirStat` instead of `length, space, inodes uint64`

Reply:
Maybe we can do the refactor first, then it will be easy to add `length`

Reply:
Done